.github/workflows/playwright.yml

```
@@ -0,0 +1,108 @@
+            const imageMarkdown = `![${screenshotImage}](https://raw.githubusercontent.com/${context.repo.owner}/${context.repo.repo}/${pr.head.ref}/public/screenshots/${screenshotImage})`
+            const commentBody = `🧪 **Playwright Snapshot Previews:**\n\n${imageMarkdown}`
+
+            // Get all comments on the PR
+            const listComments = await github.rest.issues.listComments({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: pr.number,
+            })
+
+            // Find existing bot comment
+            const existingComment = listComments.data.find(comment => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)